src/cdk/constructs/BeanstalkDistribution.ts

```diff
@@ -25,12 +26,39 @@ export class BeanstalkDistribution extends Construct {
   public readonly ebS3: s3.Bucket
   public readonly ebInstanceProfile: iam.CfnInstanceProfile
   public readonly ebInstanceProfileRole: iam.Role
+  public readonly vpc: Vpc
+  public readonly securityGroup: SecurityGroup
 
   constructor(scope: Construct, id: string, props: BeanstalkDistributionProps) {
     super(scope, id)
 
     const { stage, nodejs, isProduction, staticS3Bucket, region, appName } = props
 
+    this.vpc = new Vpc(this, 'BeanstalkVPC', {
+      natGateways: 1,
```

Comment:
We we need NAT?

Reply:
since we need internet access to external resources